mujoco_warp/_src/collision_gjk.py

```diff
@@ -155,26 +155,23 @@ def _support(geom: Geom, geomtype: int, dir: wp.vec3) -> SupportPoint:
       vert_edgeadr = geom.graphadr + 2
       vert_globalid = geom.graphadr + 2 + numvert
       edge_localid = geom.graphadr + 2 + 2 * numvert
-      # hillclimb until no change
       prev = int(-1)
-      imax = int(0)
-      if geom.index > -1:
-        imax = geom.index
-        sp.cached_index = geom.index
+      imax = wp.where(geom.index > -1, geom.index, 0)
+      sp.cached_index = geom.index
```

Comment:
you are setting `cached_index` here and then setting it again on line 175.  Seems like you don't need this line?